openedx/core/djangoapps/bookmarks/models.py

```diff
@@ -81,7 +81,7 @@ def create(cls, data):
 
             xblock_cache = XBlockCache.create({
                 'usage_key': usage_key,
-                'display_name': block.display_name,
+                'display_name': block.display_name or '',
```

Comment:
Note that you should also verify (wherever this is shown in the UI) that the display_name value is being escaped.